lang/cs/Org.Apache.REEF.Tang/Implementations/ClassHierarchy/NamedParameterNodeImpl.cs

```diff
@@ -85,5 +97,15 @@ public bool IsList()
         {
             return isList;
         }
+
+        public string GetAlias()
```

Comment:
It is defined in public interface INamedParameterNode, the implementation cannot be internal. As I mentioned before, we need to revisit INamedParameterNode to see if the entire interface should be internal.